views/coachProfile_coach.pug

```diff
@@ -1,8 +1,11 @@
 extends layout
 
 block content
+	script
+		alert("!")
```

Comment:
should we leave this alert in the code for production?

Reply:
No, I just deleted it.